addon/components/amount-input.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="amount-input" ...attributes>
+  <span class="amount-input__currency {{ if @value "black" }}">{{this.currency}}</span>
```

Comment:
```suggestion
  <span class="amount-input__currency {{if @value "black"}}">{{this.currency}}</span>
```

Reply:
Wonder why linting didn't pick this up, have updated. Will take a look at linting in another PR.